src/App/components/Dialog/index.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Icon } from "#";
+
+const Dialog = ({ diaId, diaHeader, children }) => (
+    <>
+        <div className="dialog" id={diaId} role="dialog" aria-modal="true" aria-labelledby="aria-label-dia" aria-describedby="aria-describe-dia">
+            <section>
+                <header className="dialog-header" id="aria-label-dia">
```

Comment:
Shouldn't the ID represent the element, and not the attribute referring to it?

Reply:
Yes, but this is just an internal attribute not used/seen anywhere else. `{diaId}` gets replaced with the string passed to it.